worldedit-core/src/main/java/com/fastasyncworldedit/core/command/HistorySubCommands.java

```diff
@@ -1,4 +1,4 @@
-package com.sk89q.worldedit.command;
+package com.fastasyncworldedit.core.command;
```

Comment:
I very much disagree with having command registration classes in multiple different locations. No-one's going to be using them API-wise and it just makes it more difficult to maintain. Command stuff is hard enough to traverse as it is, and as it's just not API at all, I don't think there's much need to move it.

Reply:
Every class that is 100% added by us has been moved to Fawe equivalent packages. Annotations, command registrations or anything that has `protected`/`default` access could be moved back. A `packaged-info` for affected packages could be added for class additions.

Reply:
Piston makes/requires enough of a mess as it is, I think that having FAWE command stuff is the worldedit classpath is fine as long as it's obviously so tbh

Reply:
I have to agree with @dordsor21 on this. Although we should add something in maybe a  `package-info` so we know for sure what is added by FAWE.